docs/download.md

```diff
@@ -61,38 +60,40 @@ for your hardware's minimum driver version.
 *For Cloudera and EMR support, please refer to the
 [Distributions](https://docs.nvidia.com/spark-rapids/user-guide/latest/faq.html#which-distributions-are-supported) section of the FAQ.
 
-### Download v23.08.2
+#### RAPIDS Accelerator's Support Policy for Apache Spark
```

Comment:
@mattf this section is for the support policy.
Please suggest an alternate if this description doesn't accurately convey our policy.